### PR TITLE
Update Healthcheck actuator to be compatible with ALV 

### PR DESCRIPTION
**Pull request by hananiel**
## Description

Make Health check endpoint compatible with latest version of SpringBoot endpoint to be compatible with App Live View on TAP. 

Fixes #{bug number}

## Quality checklist

- [ ] Your code complies with our [Coding Style](https://github.com/SteeltoeOSS/.github/blob/main/contributing-docs/contributing-code-style.md).
- [ ] You've updated unit and/or integration tests for your change, where applicable.
- [ ] You've updated documentation for your change, where applicable.
      If your change affects other repositories, such as [Documentation](https://github.com/SteeltoeOSS/Documentation), [Samples](https://github.com/SteeltoeOSS/Samples) and/or [MainSite](https://github.com/SteeltoeOSS/MainSite), add linked PRs here.
- [ ] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.
- [ ] You've added required license files and/or file headers (explaining where the code came from with proper attribution), where code is copied from StackOverflow, a blog, or OSS.

### CODE REVIEW

File: src/Common/src/Abstractions/Steeltoe.Common.Abstractions.csproj [hunk 1]
```diff
@@ -11,5 +11,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="$(ExtensionsVersion)" />
+    <PackageReference Include="System.Text.Json" Version="6.0.4" />
```
Comment (TimHess) [inline src/Common/src/Abstractions/Steeltoe.Common.Abstractions.csproj:14]: I'm not sure about bringing System.Text.Json and the Json attributes into this package... the abstraction is supposed to be transport agnostic. For example, would this affect potential to add support for using Spring Boot Admin over gRPC?

**Comment by TimHess** [comment]
This PR will be part of 4.0, we'll need to backport to `release/3.2` for inclusion in 3.2.1

File: src/Common/src/Abstractions/Steeltoe.Common.Abstractions.csproj (hunk 1 above)
Comment (bart-vmware) [reply src/Common/src/Abstractions/Steeltoe.Common.Abstractions.csproj:14]: Note that #873 introduces a centralized version placeholder, so please use `$(SystemVersion)` once it has been merged.

Comment (hananiel) [reply src/Common/src/Abstractions/Steeltoe.Common.Abstractions.csproj:14]: > I'm not sure about bringing System.Text.Json and the Json attributes into this package... the abstraction is supposed to be transport agnostic. For example, would this affect potential to add support for using Spring Boot Admin over gRPC?



> I'm not sure about bringing System.Text.Json and the Json attributes into this package... the abstraction is supposed to be transport agnostic. For example, would this affect potential to add support for using Spring Boot Admin over gRPC?

In 4.0, this is going to have to be completely redesigned. By exposing types that library users will need to implement health checks (ex HealthCheckResult) and statically typing it to the response of HealthEndpoint, AND using the same object for serialization, we are exposing too much to safely make changes in a backwards compatible way. It is hard to make a change without a breaking change. Considering the situation we are in my suggestion is take this dependency for now. When we do implement gRPC IMO the best thing would be to have separate packages for gRPC & Http.

**Comment by bart-vmware** [comment]
@hananiel This PR has merge conflicts. Can you update?

**Pull request closed without merging**